apps/balancer-tools/src/app/apr/pool/(components)/HistoricalAPRChart.tsx

```diff
@@ -0,0 +1,56 @@
+import { blueDark } from "@radix-ui/colors";
+
+import { trimTrailingValues } from "#/lib/utils";
+
+import { calculateAPRForPool } from "../../(utils)/calculateRoundAPR";
+import { Round } from "../../(utils)/rounds";
+import HistoricalAPRPlot from "./HistoricalAPRPlot";
+
+export default async function HistoricalAPRChart({
+  roundId,
+  poolId,
+}: {
+  roundId: string;
+  poolId: string;
+}) {
+  const HOVERTEMPLATE = "%{x}<br />%{y:.2f}% APR<extra></extra>";
+  const LAST_ROUND_ID = parseInt(Round.getAllRounds()[0].value);
+  const APRPerRoundCords: { x: string[]; y: number[] } = { x: [], y: [] };
+  for (let index = 1; index < LAST_ROUND_ID; index++) {
+    const { apr } = await calculateAPRForPool({ poolId, roundId: index });
+    APRPerRoundCords.y.push(apr);
+    APRPerRoundCords.x.push(`Round ${index}`);
+  }
+  const trimmedArray = trimTrailingValues(
+    APRPerRoundCords.x.reverse(),
+    APRPerRoundCords.y.reverse(),
+    0,
+  );
+  const APRPerRoundData = {
+    name: "APR %",
+    hovertemplate: HOVERTEMPLATE,
+    x: trimmedArray.trimmedIn,
+    y: trimmedArray.trimmedOut,
+    line: { shape: "spline" },
+    type: "scatter",
+  };
+
+  const chosenRoundMarkerIDX = APRPerRoundData.x.indexOf(`Round ${roundId}`);
```

Comment:
```suggestion
  const chosenRoundMarkerIDX = APRPerRoundData.x.findIndex(item => item === `Round ${roundId}`);
```

because `APRPerRoundData.x` is `number[] | string[]`, therefore `indexOf()` expects type never Array<T>.indexOf(searchElement: never, fromIndex: number | undefined): number